listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx

```diff
@@ -450,36 +449,48 @@ export default class UserEntityChart extends React.Component<
                   <ul className="dropdown-menu" role="menu">
                     <li>
                       <a
-                        href=""
+                        href={`?${this.buildURLParams(1, "week", entity)}`}
                         role="button"
-                        onClick={(event) => this.changeRange("week", event)}
+                        onClick={(e) => {
+                          e.preventDefault();
```

Comment:
I moved this from changeRange (and from changePage) because to me it seems odd to pass a parameter into the method when it's not really used. While this change does result in more lines of code, I think it's clearer to show that the value of the event isn't important for the changeRange method.
Happy to discuss this

Reply:
I think it could be an issue when for example we try to fix the bug we currently have with using ctrl+click to open in a new tab.

If the solution is to check a condition of the event (is ctrl key pressed?) before deciding what to do, it would be better to only have to do it in one spot.

Reply:
that's a good point - I found the control-click fix after I made this change. Keep in mind that we'd want this check in many different methods, not just `changeRange`, so I'm not sure if we should make another `handleClick` method (with callback?) or add it to each of the click handlers that we have around the app.

Reply:
I updated this PR to include support for ctrl-click